code/__HELPERS/unsorted.dm

```diff
@@ -1496,3 +1496,13 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 			else
 				return "\[[url_encode(thing.tag)]\]"
 	return "\ref[input]"
+
+/proc/turf_contents_in_typecache(var/turf/T, var/list/TYPECACHE)
+	var/list/processing_list = T.contents
```

Comment:
this can just be an atom proc. Remove the var/s, lowercase TYPECACHE and rename it contents_in_typecache